src/components/Button.js

```diff
@@ -2,16 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button({ buttonName }) {
+function Button({ buttonName, wide, color }) {
+  const styles = wide
+    ? { width: '50%' }
+    : { width: '25%' };
+
+  styles.backgroundColor = color
+    ? '#dfdfdf'
+    : '#F29444';
+
   return (
-    <button type="button" className="Button" id="button">
-      {buttonName}
-    </button>
+    <div className="button" style={styles} id="button">
+      <p>{ buttonName}</p>
+    </div>
   );
 }
 
+Button.defaultProps = {
+  wide: false,
+  color: false,
+};
+
 Button.propTypes = {
   buttonName: PropTypes.string.isRequired,
+  wide: PropTypes.bool,
+  color: PropTypes.bool,
```

Comment:
- [ ] Default color for button is orange. Color prop is a string not a boolean.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)